2-mandatory/2-oxygen-levels.js

```diff
@@ -11,7 +11,12 @@
     Some string methods that might help you here are .replace() and .substring().
 */
 
-function findSafeOxygenLevel() {}
+function findSafeOxygenLevel(array) {
```

Comment:
This solution works well. I am finding it a bit hard to read. Do you have Prettier installed?

Reply:
I cant read it  either.Appologies ! I didnt finish it and yes I have prettier installed .